src/features/locators/types/locator.types.ts

```diff
@@ -72,7 +72,7 @@ export enum LocatorValidationWarnings {
   NewElement = "The locator leads to the new element.",
 }
 
-export type LocatorValidationErrorType = LocatorValidationErrors | LocatorValidationWarnings | "";
+export type LocatorValidationErrorType = LocatorValidationErrors | LocatorValidationWarnings | string;
```

Comment:
In this case we can assign any string like a locator message, can't we? And LocatorValidationErrors and LocatorValidationWarnings doesn't make sense as types here

Reply:
yes, because we have message like this: `${length} ${LocatorValidationErrors.MultipleElements}`